src/remediations/remediations.queries.js

```diff
@@ -108,7 +108,8 @@ exports.list = function (
         },
         group: ['remediation.id'],
         order: [
-            [col(primaryOrder), asc ? 'ASC' : 'DESC'],
+            // Handle sorting by `issue_count` or `system_count` correctly
+            [col(primaryOrder === 'issue_count' ? 'issue_count' : primaryOrder === 'system_count' ? 'system_count' : `remediation.${primaryOrder}`), asc ? 'ASC' : 'DESC'],
```

Comment:
Yes this looks complicated... But joining the playbook_runs table broke the sorting so all this was necessary :/

Reply:
lolz